napari/_vispy/vispy_image_layer.py

```diff
@@ -14,6 +14,7 @@
     np.dtype(np.int16),
     np.dtype(np.uint16),
     np.dtype(np.float32),
+    np.dtype(np.float64),
```

Comment:
this helps skipping the data astype call during conversion, the label data is of dtype float64